### ISSUE TITLE
Unable to use NSwag in .Net 6 GA due to versioning conflict

### ISSUE THREAD

**Issue by maxkoshevoi** [RicoSuter/NSwag#3699]
Why this does not include 6th version? https://github.com/RicoSuter/NSwag/blob/master/src/NSwag.AspNetCore/NSwag.AspNetCore.csproj#L27

I have an `NU1107` cause of it.

```cs
Error	NU1107	Version conflict detected for Microsoft.Extensions.DependencyInjection.Abstractions. Install/reference Microsoft.Extensions.DependencyInjection.Abstractions 6.0.0 directly to project MyApi to resolve this issue. 
 MyApi -> Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore 6.0.0 -> Microsoft.EntityFrameworkCore.Relational 6.0.0 -> Microsoft.EntityFrameworkCore 6.0.0 -> Microsoft.Extensions.Logging 6.0.0 -> Microsoft.Extensions.DependencyInjection.Abstractions (>= 6.0.0) 
 MyApi -> NSwag.AspNetCore 13.13.2 -> NSwag.Generation.AspNetCore 13.13.2 -> Microsoft.Extensions.DependencyInjection.Abstractions (>= 5.0.0 && < 6.0.0).
```

And I'm unable to install `Microsoft.Extensions.DependencyInjection.Abstractions 6.0.0` cause of 
```cs
NU1608: Detected package version outside of dependency constraint: NSwag.Generation.AspNetCore 13.13.2 requires Microsoft.Extensions.DependencyInjection.Abstractions (>= 5.0.0 && < 6.0.0) but version Microsoft.Extensions.DependencyInjection.Abstractions 6.0.0 was resolved.
```

**Comment by onionhammer** [comment]
I'm seeing the same issue

**Comment by asadsahi** [comment]
And if we try and lift Microsoft packages to 6.0.0 by force i.e manually replace packages in csproj file, then the only incompatibility error that I get in my project is from NSwag, which looks like this:

![image](https://user-images.githubusercontent.com/3470892/140817818-b7aab21d-e657-40a1-a3fa-af466107f6f2.png)

So, I believe .Net 6 compatability is missing.

**Comment by syska** [comment]
No issues here when installing the package in the project to 6.0.0 ... 

Running 2 websites in production now ... so it's not a shot stopper... I guess it will be fixed within the next few days.

![image](https://user-images.githubusercontent.com/317770/140834496-18c120ef-55a9-4ba6-878a-5529224192fc.png)

I guess the maintainer will accept PRs for it.

**Comment by KostaMadorsky** [comment]
Had to add these packages into .csproj for now to make it work:
```
<PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
<PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
```

**Comment by enkodellc** [comment]
> Had to add these packages into .csproj for now to make it work:
> 
> ```
> <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
> <PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
> ```

@KostaMadorsky  that worked for me only if I put those packages in BEFORE I updated all the other packages including NSwag to .Net 6.  Thanks for the tip.

**Comment by onionhammer** [comment]
With an MSBuild target to automatically generate my API client, I'm getting an issue using NSwagExe_Net60 as well

```
error MSB3073: The command " aspnetcore2openapi /AspNetCoreEnvironment:Development /noBuild:true /project:. /output:bin\Debug\net5.0\openapi.json" exited with code 9009.
```

**Comment by onionhammer** [comment]
Getting the following with MSBuilt target as well;

```
.csproj(44,5): error MSB3073: The command " aspnetcore2openapi /AspNetCoreEnvironment:Development /noBuild:true 
/project:. /output:bin\Debug\net6.0\openapi.json" exited with code 9009.
```

**Comment by SeppPenner** [comment]
> When I say it worked I mean it worked locally but when I published to my server it failed. I do not know if it was because of NSwag or something else.

Must be something else, we have it running yet without problems (In Docker and locally).

**Comment by thestonehead** [comment]
How did you guys solve the problem of missing "Net60 release"  and .Net 6 NSwag binaries when having it as a build-step?

e.g.
`  <Target Name="NSwag" AfterTargets="Build">
    <Copy SourceFiles="@(Reference)" DestinationFolder="$(OutDir)References" />
    <Exec Command="$(NSwagExe_Net50) run nswag.json /variables:Configuration=$(Configuration),OutDir=$(OutDir) /runtime:Net50" />
    <RemoveDir Directories="$(OutDir)References" />
  </Target>`

**Comment by SeppPenner** [comment]
Should be fixed with https://github.com/RicoSuter/NSwag/pull/3632.

**Comment by luizfbicalho** [comment]
Just waiting for this to update my project to 6.0

**Comment by RicoSuter** [comment]
NSwag with .NET 6 support is released: Try v13.14.0

**Comment by SeppPenner** [comment]
I can confirm that the issues don't occur anymore :)
